phoenix-queryserver/pom.xml

```diff
@@ -147,6 +147,14 @@
       <groupId>commons-logging</groupId>
       <artifactId>commons-logging</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.phoenix</groupId>
+      <artifactId>phoenix-queryserver-loadbalancer</artifactId>
```

Comment:
I don't understand how that is the "same issue". PQS doesn't depend on the load balancer; the load-balancer should depend on PQS (for the interface).